src/components/editor/markdown-renderer/markdown-renderer.scss

```diff
@@ -14,6 +14,11 @@
     max-width: 900px;
   }
 
+  & > img {
+    width: unset;
```

Comment:
As I'm currently unable to test it (being not at home): Did you test that this statement does not interfere with the wider-images feature in view-only mode?

Reply:
Seems to be alright. I have also created a PR at the `markdown-it-plantuml` repo to be able to use a class name selector for this. The maintainer hasn't responded so far. https://github.com/gmunguia/markdown-it-plantuml/pull/24

Reply:
This should be fine, because the normal images are wrapped in p Tags that have the wider class